Tests/JavaScriptVisitTests.swift

```diff
@@ -0,0 +1,3 @@
+import XCTest
+
+class JavaScriptVisitTests: XCTestCase {}
```

Comment:
As a relative newcomer to the repo, is there any context to this empty test file, or does it serve as a big `TODO` 😅?

Reply:
The file existed in the Quick/Nimble spec version like this. So I just copied it over.

Happy to remove it!

Reply:
Removed.